server/src/main/java/io/kroki/server/Server.java

```diff
@@ -68,7 +69,14 @@ public void start(Promise<Void> startPromise) {
   }
 
   static void start(Vertx vertx, JsonObject config, Handler<AsyncResult<HttpServer>> listenHandler) {
-    HttpServer server = vertx.createHttpServer();
+    Integer maxInitialLineLength = config.getInteger("KROKI_MAX_LINE_LENGTH");
```

Comment:
I think we should use the name `KROKI_MAX_URI_LENGTH`.
In my opinion, max line is a bit unclear.

Reply:
Agree